assets/js/lib/test-utils/factories/executions.js

```diff
@@ -17,15 +17,30 @@ export const executionValueFactory = Factory.define(({ sequence }) => ({
 export const executionExpectationEvaluationFactory = Factory.define(
   ({ sequence, params }) => {
     const name = params.name || `expectation_${sequence}`;
+    const type = params.type || expectationReturnTypeEnum();
```

Comment:
I don't think you need these changes (except the failure message one).
The returned object values take what it comes from `params` automatically, even if you don't define them here, using the default value you define there.

I mean, leaving:
```
return {
  name,
  return_value: faker.datatype.number(),
  type: expectationReturnTypeEnum()
...
```
Achieves the same without adding the top lines with the param.

And if you want to have to coming from params, you can do something like this:
```
{
   name = `expectation_${sequence}`,
   failure_message = params.failure_message ? { failure_message: params.failure_message } : {}
} = params;
```

https://github.com/thoughtbot/fishery#use-params-to-access-passed-in-properties

Reply:
Thanks for the reminder :smile: